src/Pages/EunsuLim/Login/Login.js

```diff
@@ -1,21 +1,188 @@
-import React from 'react'
-import './Login.scss'
-import '../../../styles/reset.scss'
-import '../../../styles/common.scss'
-import { withRouter } from 'react-router-dom';
+import React from "react";
+import { withRouter } from "react-router-dom";
+import "../../../styles/reset.scss";
+import "../../../styles/common.scss";
+import "./Login.scss";
 
+const footerItem = [
```

Comment:
변하는 않는 데이터는 대문자_대문자 컨벤션 적용해주세요!
```suggestion
const FOOTER_ITEM= [
```

Reply:
변경했습니다.